binary_field/fields.py

```diff
@@ -271,23 +271,6 @@ def _fnct_write(self, obj, cr, uid, ids, field_name, value, args,
                         obj, cr, uid, ids, name, context=context)
         return True
 
-    def _read_binary(self, storage, record, field_name, binary_uid,
```

Comment:
Could you explain (in the PR description?) why you remove this function?
Else it is hard for reviewers to figure out if this makes sense :)